microsoft/testsuites/vm_extensions/vmsnapshot_extension.py

```diff
@@ -100,6 +129,34 @@ def verify_vmsnapshot_extension(
                     pass
                 else:
                     raise e
-            time.sleep(1)
+            time.sleep(60)
             count = count + 1
         assert_that(count, "Restore point creation failed.").is_less_than(10)
+
+    def get_restore_point(
+        self, log: Logger, node: Node, environment: Environment,
+        resource_group_name: str, restore_point_collection: str,
+        restore_point: str
+    ) -> int:
+        assert environment.platform
+        platform: AzurePlatform = environment.platform  # type: ignore
+        assert isinstance(platform, AzurePlatform)
+        client = get_compute_client(platform)
+        response = client.restore_points.get(
+            resource_group_name=resource_group_name,
+            restore_point_collection_name=restore_point_collection,
+            restore_point_name=restore_point,
+            expand=None)
+        if response.provisioning_state == "Succeeded":
+            log.info(f"restore point {restore_point} created")
+            consistency_mode = response.consistency_mode
+            log.info(
```

Comment:
nit: line breaks appear inconsistent. line 161 has all logged content in single line but this line has line breaks. Simi,ar observation for L120, 117 and others.

Reply:
this was done to ensure that the length of the line is not exceeding certain characters, ig 83